sim/core/aura.go

```diff
@@ -55,6 +55,8 @@ var RangedSwingCooldownID = NewCooldownID()
 var OffensiveTrinketSharedCooldownID = NewCooldownID()
 var DefensiveTrinketSharedCooldownID = NewCooldownID()
 
+type OnExpire func(sim *Simulation)
```

Comment:
such a stupid thing - and you can delay this until later if not a big deal, can you move this to be with the other defs of 'OnCast' and 'OnCastComplete' etc. Makes it easier to find them all

Reply:
Talked offline, going to just submit this for now but I don't feel that strongly. Also its easy to move later, won't break anything.